assets/js/index.js

```diff
@@ -0,0 +1,244 @@
+    options: ["var", "let", "Both var and let", "none"],
+    correctAnswer: "Both var and let",
+  },
+  {
+    text: "Inside which HTML element do we put the JavaScript?",
+    options: ["<js>", "<javascript>", "<script>"],
+    correctAnswer: "<script>",
+  },
+  {
+    text: "Which of the following will write the message “Hello DataFlair!” in an alert box?",
+    options: [
+      " alertBox(“Hello DataFlair!”)",
+      "alert(Hello DataFlair!)",
+      " alert(“Hello DataFlair!”)",
+    ],
+    correctAnswer: " alert(“Hello DataFlair!”)",
+  },
+  {
+    text: "Which of the following statements will throw an error?",
+    options: [
+      "  var fun = function bar( ){ }",
+      "var fun = function bar{ }",
+      "  function fun( ){ }",
+    ],
+    correctAnswer: "var fun = function bar{ }",
+  },
+  {
+    text: "Which are the correct “if” statements to execute certain code if “x” is equal to 2?",
+    options: ["  if(x = 2)", " if(x == 2)", " if(x 2)"],
+    correctAnswer: "  if(x == 2)",
+  },
+];
+
+let questionIndex = 0;
+let timer = 100 * questions.length;
+
+const startButton = document.getElementById("start-quiz-button");
+const main = document.getElementById("main");
+const timerSpan = document.getElementById("timer-span");
+const formElement = document.getElementById("submit-score-button");
+
+let arrayFromLS = JSON.parse(localStorage.getItem("feedbackResults"));
+
+if (!arrayFromLS) {
+  localStorage.setItem("feedbackResults", JSON.stringify([]));
+}
+
+//function to record click on available options.
+
+//Game Over
+const gameOver = () => {
+  section = document.createElement("game-over-section");
+
+  h2 = document.createElement("h2");
+  h2.setAttribute("class", "high-scores-value");
+  h2.textContent = "Game Over!";
+
+  h3 = document.createElement("h3");
+  h3.setAttribute("class", "game-over-message");
+  h3.textContent = "Please select home  to retake  quiz";
+
+  section.append(h2, h3);
+  main.append(section);
+};
+//Remove current Question
+
+//Store values in LS
+const storeInLS = (key, value) => {
+  if (!arrayFromLS) {
+    arrayFromLS = [];
+  }
+  arrayFromLS.push(value);
+  localStorage.setItem(key, JSON.stringify(arrayFromLS));
+};
+
+const displayResults = (event) => {
+  event.preventDefault();
+  //get initials from form
+  const initials = document.getElementById("name-initials");
+  if (initials) {
+    const yourScore = {
+      initials: initials.value,
+      score: timer,
+    };
+    storeInLS("feedbackResults", yourScore);
+    initials.value = "";
+  } else {
+    alert("Please enter your initials");
+  }
+};
+//Timer Function called
+const setTimer = () => {
+  const updateTimerValue = () => {
+    //start timer and decrement by 1
+
+    timer -= 1;
+
+    //set text
+    timerSpan.textContent = timer;
+
+    // check if timer is equal to 0
+    if (timer <= 0) {
+      clearInterval(timerId);
+
+      //Delete Timer
+      const removeTimer = document.getElementById("timer-section");
```

Comment:
Think about the names of your variable. This is just the timer section, so name it like `const timerSection`